project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("com.twitter" % "scrooge-sbt-plugin" % "19.11.0")
+addSbtPlugin("com.twitter" % "scrooge-sbt-plugin" % "21.8.0")
```

Comment:
I found that this version needs to match the `"com.twitter" %% "scrooge-core" % "21.8.0"`